CHANGELOG.rst

```diff
@@ -28,7 +28,8 @@ Change Log
     - A adjudicator/team conflicts with an adjudicator if *any* institution appears as an institutional conflict for both parties
 - When printing scoresheets you can now edit the motions display just on that printing page. This allows you to use placeholder motions in Tabbycat (in order to prevent leaks) while still producing ballots with the correct motions
 - Info-slides can now be split into paragraphs
-- Instiututional caps in breaks can be based on the number of teams in the break. Thanks to Viran Weerasekera for this feature!
+- Check-in pages now differentiate between teams with 1 and 2 checked-in people in two-team formats
```

Comment:
Not just two-team formats!

Reply:
Oh, are there four-team formats with 3+ speakers per team?

Reply:
The two comments were unrelated. For the changelog, I'm noting that there isn't an entry about the different styles in checkins.